configure.ml

```diff
@@ -388,10 +393,10 @@ let reset_caml_find c o = c.find <- o
 
 let coq_debug_flag = if !Prefs.debug then "-g" else ""
 let coq_profile_flag = if !Prefs.profile then "-p" else ""
-let coq_annotate_flag =
-  if !Prefs.annotate
-  then if program_in_path "ocamlmerlin" then "-bin-annot" else "-annot"
-  else ""
+let coq_annotate_flags =
+  String.trim
+    ((if !Prefs.bin_annotate then "-bin-annot" else "") ^
+       " " ^ (if !Prefs.annotate then "-annot" else ""))
```

Comment:
The right way to do this in OCaml is to use `sprintf`, not `trim`, but anyways this fake flag should be removed and the proper flag added below instead.